src/event.js

```diff
@@ -5,7 +5,22 @@ export const customEvents = popup => {
 			const toElement = e.toElement || e.relatedTarget || e.target
 			if ( toElement !== popup.element.currentTargetElement &&
 			!popup.element.contains( toElement ) ) {
-				popup.hide()
+				let timeoutId
+				const previewElement = popup.element.currentTargetElement,
```

Comment:
`popup.element.currentTargetElement` used twice within the same scope, let's define `previewElement` in the top of the scope.